pkg/scheduler/framework/plugins/volumerestrictions/volume_restrictions_test.go

```diff
@@ -252,47 +273,68 @@ func TestAccessModeConflicts(t *testing.T) {
 	tests := []struct {
 		name                   string
 		pod                    *v1.Pod
+		nodeInfo               *framework.NodeInfo
 		existingPods           []*v1.Pod
 		existingNodes          []*v1.Node
 		existingPVCs           []*v1.PersistentVolumeClaim
 		enableReadWriteOncePod bool
+		preFilterWantStatus    *framework.Status
 		wantStatus             *framework.Status
 	}{
 		{
 			name:                   "nothing",
 			pod:                    &v1.Pod{},
+			nodeInfo:               framework.NewNodeInfo(),
 			existingPods:           []*v1.Pod{},
 			existingNodes:          []*v1.Node{},
 			existingPVCs:           []*v1.PersistentVolumeClaim{},
 			enableReadWriteOncePod: true,
+			preFilterWantStatus:    nil,
 			wantStatus:             nil,
 		},
 		{
 			name:                   "failed to get PVC",
-			pod:                    podWithReadWriteOncePodPVC,
+			pod:                    podWithOnePVC,
+			nodeInfo:               framework.NewNodeInfo(),
 			existingPods:           []*v1.Pod{},
 			existingNodes:          []*v1.Node{},
 			existingPVCs:           []*v1.PersistentVolumeClaim{},
 			enableReadWriteOncePod: true,
-			wantStatus:             framework.NewStatus(framework.UnschedulableAndUnresolvable, "persistentvolumeclaim \"claim-with-rwop\" not found"),
+			preFilterWantStatus:    framework.NewStatus(framework.UnschedulableAndUnresolvable, "persistentvolumeclaim \"claim-with-rwop-1\" not found"),
+			wantStatus:             framework.AsStatus(ErrStatusKeyNotFound),
```

Comment:
Actually, what happens in the scheduler when preFilter returns UnschedulableAndUnresolvable?

IIUC, Filter wouldn't run. So having Filter run in the test is kind of useless, isn't it?

Reply:
Updated to only run the `wantStatus` test if `preFilterWantStatus` is successful, which allowed me to get rid of `ErrStatusKeyNotFound`